tests/test_entities.py

```diff
@@ -100,6 +101,7 @@ def test_self_path_with_which(self, entity, path, which):
     @data(
         (entities.ConfigTemplate, '/config_templates', 'build_pxe_default'),
         (entities.ConfigTemplate, '/config_templates', 'revision'),
+        (entities.RHCIDeployment, '/deployments', 'deploy'),
```

Comment:
Unfortunately, this won't work. This test basically does the following:

    path = entities.RHCIDeployment().path()

It then makes some assertions about `path`. That won't work here, given that the "deploy" path is only valid when an ID is provided.